projects/dotcms-field-elements/src/components/dot-tags/dot-tags.e2e.ts

```diff
@@ -10,15 +10,16 @@ describe('dot-tags', () => {
 
     const getAutoComplete = () => page.find('dot-autocomplete');
     const getChips = () => page.findAll('dot-chip');
-
-    beforeEach(async () => {
+    const createEmptyDotTags = async () => {
         page = await newE2EPage();
         await page.setContent(`<dot-tags></dot-tags>`);
         element = await page.find('dot-tags');
         await page.waitForChanges();
-    });
+    };
 
     describe('css classes', () => {
+        beforeEach(async () => await createEmptyDotTags());
```

Comment:
```suggestion
        beforeEach(async () => {
            await createEmptyDotTags()
        });
```

This is returning the execution of the function, probably is not breaking anything but not sure if it has side effects.